### DISCUSSION TITLE
Check Boxes on time blocks are cosmetic

### DISCUSSION THREAD

**Discussion by intervolver** [ivan-lednev/obsidian-day-planner#446] [Features & bugs]
Several features that would be nice:
- Allow checking off a task in the planner itself
- a setting so that newly created blocks do not default to checkboxes
- the ability to enter text directly into the newly created time block, similar to the way kanban boards allow direct entry into new cards

**Comment by Xelgod** [comment]
Yeah I was very confused as on hovering the checkboxes an interaction cursor pops up but they are not clickable.

**Comment by ivan-lednev** [comment]
The issue in the title is a duplicate of #304. Please create separate tickets for separate feature requests so as not to confuse them.
